src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue

```diff
@@ -1,26 +1,125 @@
 <template>
-  <div>
+  <div class="container-max-width">
     <h1>
       Ready to submit your acquisition package?
     </h1>
-    <div class="copy-max-width">
-      <p class="mt-2 mb-4">
-        Future ready to submit
-      </p>
+    <div class="d-flex mt-10">
+      <div class="copy-max-width">
+        <p class="font-size-20 font-weight-500 mb-3">
+          Great news! We have everything ready to send your package to DITCO for processing.
+        </p>
+        <p class="mt-2 mb-10">
+          Upon submission, a Contracting Specialist will review your documents and
+          contact you if any changes are required. You’ll continue to have access
+          to your completed documents within DAPPS.
+        </p>
+        <div  class="border1 border-rounded-more border-base-lighter pa-6 bg-primary-lighter">
+          <ATATCheckboxGroup
+            id="CertifiedCheckbox"
+            aria-describedby="CertifiedCheckbox"
+            :value.sync="certified"
+            :items="checkboxItem"
+            :boldText="true"
+            :card="false"
+          />
+          <ol class="ml-6">
+            <li class="pb-3">
+              All information in my package is accurate and complete to the best of my knowledge.
+            </li>
+            <li class="pb-3">
+              All documents requiring certification have been signed and uploaded.
+            </li>
+          </ol>
+        </div>
+      </div>
+      <div class="ml-10">
+        <v-card
+          class="
+            border1
+            border-base-lighter
+            pa-4"
+          :elevation="2"
+        >
+          <h3 class="mb-2">Your completed package includes:</h3>
+          <ul>
+            <li
+              v-for="(item,idx) in documentList"
+              :key="idx"
+              class="text-base py-1"
+            >
+              {{item}}
+            </li>
+          </ul>
+          <v-btn
+            class="secondary _text-decoration-none px-6 mt-3"
+            large
+            role="button"
+            :href="downloadPackageLink"
```

Comment:
sonar cloud is complaining about security hot spot on this - look into using vue sanitize on this link perhaps? we also have a plugin in `plugins/sanitize.ts`